test/unit.test.js

```diff
@@ -82,6 +83,24 @@ for (const { testName, isRoot } of unitTests) {
   const unitPath = join(__dirname, 'unit', testName);
 
   it(`should correctly trace ${testSuffix}`, async () => {
+    // Mock Node.js version for module-sync-condition tests
+    if (testName === 'module-sync-condition') {
```

Comment:
Instead of mocking, lets just skip for older versions of node since we have a ci matrix already.

See the `continue` statements above.

Reply:
Now that I think about it, we need this first:

- https://github.com/vercel/nft/pull/535